docs/references/neuron-wallet-guide.md

```diff
@@ -105,17 +105,19 @@ You should have the third party wallet/exchange destination address. **Please ma
 
 ## Troubleshooting
 
-### Neuron displays 0 balance
+### Sync failed. XXX /Sync is slow XXX
+
+1. Quit and restart app several times.
+
+Note: Neuron requires [VC++ redistributable](https://support.microsoft.com/en-us/help/2977003/the-latest-supported-visual-c-downloads) on Windows to work properly. 
```

Comment:
Actually the VC++ redistributable is required by running CKB node, so it's better to make it more clear like "The Neuron bundled CKB requires ..."